recipe/meta.yaml

```diff
@@ -21,23 +13,25 @@ source:
     sha256: 9b469f3a900bf28dc19b8cfbf8019bf47f7fdd1a65a1d4ffb98fc14166beb4d1
     folder: certifi
   # Download bootstrapped wheels into their respective source directories
-  {%- for w in bootstrapped_wheels %}
-  - url: https://pypi.io/packages/py3/{{ w["name"][0] }}/{{ w["name"] }}/{{ w["name"] }}-{{ w["version"] }}-py3-none-any.whl
-    sha256: {{ w["sha256"] }}
-    folder: {{ w["name"] }}_wheel
-  {%- endfor %}
+  - url: https://pypi.io/packages/py3/p/pip/pip-{{ pip_version }}-py3-none-any.whl
+    sha256: 5052d7889c1f9d05224cd41741acb7c5d6fa735ab34e339624a614eaaa7e7d76
+    folder: boostrap_wheels
+  - url: https://pypi.io/packages/py3/s/setuptools/setuptools-{{ setuptools_version }}-py3-none-any.whl
+    sha256: 385eb4edd9c9d5c17540511303e39a147ce2fc04bc55289c322b9e5904fe2c05
+    folder: boostrap_wheels
+  - url: https://pypi.io/packages/py3/w/wheel/wheel-{{ wheel_version }}-py3-none-any.whl
+    sha256: 177f9c9b0d45c47873b619f5b650346d632cdc35fb5e4d25058e09c9e581433d
+    folder: boostrap_wheels
 
 build:
-  number: 1
+  number: 2
   noarch: python
   script_env:
     # Add bootstrapped wheels to `PYTHONPATH` so Python can find them for the build
     - >-
       PYTHONPATH=
-      {%- for w in bootstrapped_wheels -%}
-      {{ os.sep.join([SRC_DIR, w["name"] ~ "_wheel"]) ~ os.pathsep }}
-      {%- endfor -%}
-      {{ os.environ.get("PYTHONPATH", "") }}
+      {{- os.sep.join([SRC_DIR|default("."), "boostrap_wheels"]) ~ os.pathsep }}
+      {{- os.environ.get("PYTHONPATH", "") }}
```

Comment:
The linter is complaining about the `-` next to `{{`, which is just how Jinja strips blank space. Think the linter needs an update here